Tests/Unit/Helpers.swift

```diff
@@ -245,3 +245,11 @@ func deleteMockMiniApp(appId: String, versionId: String) {
 func deleteStatusPreferences() {
```

Comment:
The file should be refactored for smaller specific units of functionality. What do you think @Climbatize @leorjoseph ?

Reply:
We use this method in tearDown/afterEach/beforeEach process in which it deletes preferences related to a specific mini-app. As of now, we were not in a need to handle them in different units.

Reply:
Do you mean to say it is a `MiniAppPrefsRemover`?

Reply:
Yeah, something like that.